seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -45,6 +46,66 @@
 
 TEMP_PATH = '/temp_path/temp_filename'
 
+TEST_GUID=f'P_{TEST_NO_PROJECT_WORKSPACE_NAME}'
+MOCK_TOKEN = 'mock_openid_bearer'
```

Comment:
our security linter is being annoying at this one. Just add the comment `# nosec` to the end oof this line